dev-system/resources/kms_examples/update.sh

```
@@ -1,119 +1,136 @@
-    v) version="${OPTARG}" ;;             # set version and continue
-    w) manual_mode="yes" ; work ;;        # work() now and not latter, CONTINUE
+    a) additional_flags="${OPTARG}" ;;   # set flags and continue
+    e) env="${OPTARG}" ;;                # set env and continue
+    h) help ; exit ;;                    # print help and exit
+    o) output_dir="${OPTARG}" ;;         # set output dir and continue
+    p) manual_mode="yes" ; promote ;;   # promote _full to short names, CONTINUE
+    s) schemes_to_import="${OPTARG}" ;;  # set schemes and continue
+    S) stream_mode="on" ;;               # stream to stdout
+    v) version="${OPTARG}" ;;            # set version and continue
+    w) manual_mode="yes" ; work ;;       # work() now and not later, CONTINUE
   esac
```

Shellcheck (0.11.0)

[warning] Invalid flags are not handled. Add a *) case.

(SC2220)